app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/meetingServices/MeetingPhotoServiceTest.kt

```diff
@@ -303,7 +319,7 @@ class MeetingPhotoServiceTest {
             runTest {
                 MeetingPhotoServiceImpl.removeMeetingPhoto(
                     meetingWithPhotos.meetingId!!,
-                    PhotoMetadata(pictureId = " ")
+                    TrashPhotoMetadata(pictureId = " ", takenBy = null)
```

Comment:
Well spotted ! I removed it but I kept it in the following test since it directly deals with the null `takenBy` so its more explicit :

`addMeetingPhotoNullTakenByShouldThrowIllegalArgumentException`